packages/mobile/src/screens/track-screen/TrackScreen.tsx

```diff
@@ -125,7 +129,7 @@ export const TrackScreen = () => {
       <ScreenContent isOfflineCapable={isOfflineModeEnabled}>
         <Lineup
           actions={tracksActions}
-          count={6}
+          count={isReachable ? 6 : 0}
```

Comment:
Agreed it is quite hacky but only as a consequence of tech debt, maybe add a comment here to explain this